Dockerfile

```diff
@@ -2,6 +2,8 @@ ARG ROS_VERSION=foxy
 
 FROM ros2_ws:${ROS_VERSION}
 
+RUN sudo curl http://repo.ros2.org/repos.key | sudo apt-key add -
```

Comment:
This line is temporary. It fixes a current issue with ROS having its key being revoked. I think we can keep it there for now and should remove it in later versions